com.toolstack.Folio.json

```diff
@@ -0,0 +1,47 @@
+{
+    "app-id" : "com.toolstack.Folio",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "45",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "com.toolstack.Folio",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=host"
```

Comment:
this is a very broad filesystem permission. is it necessary?

Reply:
Yes, the user might want to change where their notes are stored.  Might be able to limit it to user directory though.